frontend/public/components/catalog/catalog-items.jsx

```diff
@@ -1,13 +1,13 @@
 import * as React from 'react';
 import * as _ from 'lodash-es';
 import * as PropTypes from 'prop-types';
-import {CatalogTile} from 'patternfly-react-extensions';
-import {Modal} from 'patternfly-react';
+import { CatalogTile } from 'patternfly-react-extensions';
+import { Modal } from 'patternfly-react';
```

Comment:
Can we add this format rule to eslint?

Reply:
Yeah, I was planning on adding some more. I wanted to wait for the PF4 PR, though, to avoid a lot of rebasing.